database/seeds/SourceServiceTableSeeder.php

```diff
@@ -319,6 +364,33 @@ private function addArkVariables()
         ]);
     }
 
+    private function addCSGOVariables()
+    {
+        ServiceVariable::updateOrCreate([
+            'option_id' => $this->option['csgo']->id,
+            'env_variable' => 'SRCDS_MAP',
+        ], [
+            'name' => 'Map',
+            'description' => 'The default map for the server.',
+            'default_value' => '',
+            'user_viewable' => 1,
+            'user_editable' => 1,
+            'rules' => 'required',
```

Comment:
Most likely this should be `required|string|alpha_dash` to allow `map_name` and similar. Are there names that would violate these rules?